src/main/java/Nocturne.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Scanner;
+
+public class Nocturne {
+    public static void main(String[] args) throws NocturneException {
+        Task[] tasks = new Task[100];
```

Comment:
just a recommendation: perhaps you might want to put the number 100 as a named constant to avoid magic numbers! it will also be easier for you to find & change this number in the future if everything is put on top :]